language/tools/move-cli/src/sandbox/utils/mod.rs

```diff
@@ -82,6 +83,19 @@ pub(crate) fn explain_publish_changeset(changeset: &ChangeSet, state: &OnDiskSta
     )
 }
 
+// Print a struct with a specified outer indent
+fn print_struct_with_indent(value: &AnnotatedMoveStruct, indent: u64) {
+    let mut indent_str: String = "".to_owned();
+    for _i in 0..indent {
+        indent_str.push(' ');
+    }
```

Comment:
I think this might be simplified to `(0..indent).map(|_| " ").collect::<String>();`

Reply:
Good idea; I'll fix that.